app/controllers/OneOffContributions.scala

```diff
@@ -0,0 +1,65 @@
+package controllers
+
+import actions.{CacheControl, CustomActionBuilders}
+import assets.AssetsResolver
+import play.api.mvc._
+import play.api.libs.circe.Circe
+
+import scala.concurrent.ExecutionContext
+import services.{IdentityService, TestUserService}
+import com.typesafe.scalalogging.LazyLogging
+import views.html.oneOffContributions
+import config.StripeConfigProvider
+import cats.implicits._
+import com.gu.googleauth.AuthAction
+import com.gu.identity.play.IdUser
+import models.Autofill
+import io.circe.syntax._
+
+class OneOffContributions(
+    val assets: AssetsResolver,
+    actionRefiners: CustomActionBuilders,
+    identityService: IdentityService,
+    testUsers: TestUserService,
+    stripeConfigProvider: StripeConfigProvider,
+    authAction: AuthAction[AnyContent],
+    components: ControllerComponents
+)(implicit val exec: ExecutionContext) extends AbstractController(components) with Circe with LazyLogging {
+
+  import actionRefiners._
+
+  implicit val ar = assets
+
+  def displayForm: Action[AnyContent] = CachedAction() {
+    form(uatMode = false)
```

Comment:
@davidfurey Why this is hard-coded to false? This cause that we are not able to show the test banner if it is a test user.